development/tsdb-blocks-storage-s3-single-binary/dev.dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM alpine:3.15.0
+FROM alpine:3.15.4
 
-RUN     mkdir /mimr
+RUN     mkdir /mimir
 WORKDIR /mimir
-ADD     ./mimir ./
+COPY     ./mimir ./
```

Comment:
`COPY` is preferable to `ADD`, since the latter is more complex in that it can also copy remote sources etc.